.github/workflows/_extension_deploy_wasm.yml

```diff
@@ -0,0 +1,122 @@
+#
+# Reusable workflow that deploys the artifacts produced by github.com/duckdb/duckdb/.github/workflows/_extension_distribution.yml
```

Comment:
Is there a reason this is a separate workflow? Shouldn't this just be added to the already existing `_extension_deploy.yml` ?

Reply:
I will merge the two, makes sense and it's cleaner.